code/modules/unit_tests/_unit_tests.dm

```diff
@@ -27,6 +27,7 @@
 #include "subsystem_init.dm"
 #include "surgeries.dm"
 #include "timer_sanity.dm"
+#include "test_wounds.dm"
```

Comment:
I'd rename this to just `wounds.dm`. Everything in the tests folder is going to be a test.

Reply:
yeah but having it show up as the first or second result when i search for files named "wounds" rather than the actual _DEFINES or class definition files would drive me insane

Reply:
Could change it to `wounds_sanity.dm` similar to `timer_sanity.dm`. Just make sure to preserve alphabetical order.